app/views/user/_tracks_progress.erb

```diff
@@ -0,0 +1,22 @@
+<% if user_tracks_summary.any? %>
+  <h3> Progess </h3>
+  <% if params[:all] == 'true' %>
```

Comment:
I think the default should be collapsed. Most people have few languages, and it's annoying to show all of them for the people who have lots of them.

By having it be collapsed by default, most people won't notice, and the people who have lots will be happier.